checker/tests/src/tests/NullnessRawTypesTest.java

```diff
@@ -0,0 +1,26 @@
+package tests;
+
+import org.checkerframework.framework.test.CheckerFrameworkTest;
+
+import java.io.File;
+
+import org.junit.runners.Parameterized.Parameters;
+
+/**
+ * Tests for the Nullness Checker on raw types
+ */
+public class NullnessRawTypesTest extends CheckerFrameworkTest {
+
+    public NullnessRawTypesTest(File testFile) {
+        super(testFile,
+                org.checkerframework.checker.nullness.NullnessChecker.class,
+                "nullness",
+                "-Anomsgtext");
```

Comment:
This test suite doesn't pass any special compiler flags. Is it worth adding yet another top-level nullness test directory for these three files?
By moving to the normal nullness/rawness directory, you would also get both FbC and rawness coverage.